apptainer/petsc.def

```diff
@@ -90,6 +90,8 @@ From: {{ APPTAINER_FROM }}
 %files
     {{ MOOSE_DIR }}/scripts/configure_petsc.sh {{ ROOT_BUILD_DIR }}/scripts/configure_petsc.sh
     {{ MOOSE_DIR }}/scripts/update_and_rebuild_petsc.sh {{ ROOT_BUILD_DIR }}/scripts/update_and_rebuild_petsc.sh
+    {{ MOOSE_DIR }}/scripts/apple-silicon-hdf5-autogen.patch {{ ROOT_BUILD_DIR }}/scripts/apple-silicon-hdf5-autogen.patch
+    {{ MOOSE_DIR }}/scripts/tao_restore_viewer.patch {{ ROOT_BUILD_DIR }}/scripts/tao_restore_viewer.patch
```

Comment:
I think both of these should be added to the influential files

Reply:
@lindsayad do you know how to do this? It's easy peasy. Just add the paths to the `moose/scripts/versioner.yaml` file under the appropriate 'influential' file list for petsc.